docs/reference/commandline/dockerd.md

```diff
@@ -40,6 +40,7 @@ Options:
   -D, --debug                                 Enable debug mode
       --default-gateway ip                    Container default gateway IPv4 address
       --default-gateway-v6 ip                 Container default gateway IPv6 address
+      --default-address-pools                 Set the default address pools for local node networks
```

Comment:
This should be singular now

Also, I think this page has an example `daemon.json`; can you update that one as well (with some example values)